src/include/rados/librados.hpp

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 #include "memory.h"
 #include "buffer.h"
+#include "common/ceph_time.h"
```

Comment:
we can use struct timespec from time.h... there are to/from methods in the ceph_time classes.